ivy/functional/frontends/numpy/statistics/averages_and_variances.py

```diff
@@ -158,7 +150,7 @@ def cov(
             # if w exists, use weighted average
             xw = x.multiply(w)
             w_sum = ivy.sum(w)
-            average = ivy.stable_divide(ivy.sum(xw, axis=1) , w_sum)
+            average = ivy.stable_divide(ivy.sum(xw, axis=1), w_sum)
```

Comment:
There is no need to change another function in your PR. If you want to fix other things than adding nanvar, please create separate PR

Reply:
> There is no need to change another function in your PR. If you want to fix other things than adding nanvar, please create separate PR

This happened automatically due to the pre-commit framework